### PR TITLE
Send async messages when student created, edited, deleted

### PR DESCRIPTION
**Pull request by nuzurie**

### CODE REVIEW

**Review by car12lin12 — COMMENTED** [review]
is there supposed to be a consumer? edit: my bad I see it in chat

File: app/server.go [hunk 1]
```diff
@@ -40,19 +41,46 @@ func Server(
...
+	)
+	failOnError(err, "can't create exchange")
+	mm := usecase.NewMessagingManager(ch)
 	studentRepository := repository.NewStudentRepository(pool)
 	reviewRepository := repository4.NewReviewRepository(pool)
-	studentUseCase := usecase.NewStudentUseCase(studentRepository, reviewRepository, time.Second)
+	studentUseCase := usecase.NewStudentUseCase(mm, studentRepository, reviewRepository, time.Second)
 	studentHandler := http.NewStudentHandler(studentUseCase)
-
+	go studentUseCase.CreateStudentTopic()
```
Comment (car12lin12) [inline app/server.go:82]: what is the purpose for sending (create, update, delete) messages here? I thought they would be called in the usecase or repo

Comment (nuzurie) [reply app/server.go:82]: It's starts an infinite loop of sending async messages to rabbitmq. This method is called just once here, and no where else. Only one method run is needed for the entire app.

**Comment by car12lin12** [comment]
can you .ignore the .idea folder?

**Review by car12lin12 — APPROVED** [review]
looks good!

**Pull request merged**